app/validators/keywords_form_validator.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class KeywordsFormValidator < ActiveModel::Validator
```

Comment:
I did this instead

```
class KeywordsFormValidator < ActiveModel::Validator
  attr_reader :keywords_form

  def validate(keywords_form)
    keywords_file = keywords_form.file

    keywords_form.errors.add(:base, I18n.t('keyword.file_cannot_be_blank')) if keywords_file.blank?
    keywords_form.errors.add(:base, I18n.t('csv.upload_fail_invalid_extention')) unless keywords_file.content_type == 'text/csv'
    keywords_form.errors.add(:base, I18n.t('csv.upload_fail_invalid_size')) unless CSV.read(keywords_file.path).count.between?(1, 1000)
  end
end
```

I'm having trouble getting it to work with `::EachValidator` 😬 and I seem to be stuck on it 😖 @Lahphim

Reply:
I think it's OK with the current one 🙌
Because you didn't use the option in the validator
For using `EachValidator`, it will iterate through the option hash something like

```
validates :mobile_number, presence: true, uniqueness: true, **mobile_number: { type: :simple_format }**
```